test/data/contains.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
Since we are talking of just three documents, I think that adding a new .json file sound a bit overkilled.
Could you add the three document manually at the beginning of the test?

Reply:
Like this?

```
vals := [][]int{
			{
				1, 2, 4,
			},
			{
				5, 6, 7,
			},
			{
				4, 10, 20,
			},
		}
		docs := make([]*c.Document, 3)
		for _, val := range vals {
			doc := c.NewDocument()
			doc.Set("myField", val)
			docs = append(docs, doc)
		}
		require.NoError(t, db.Insert("myCollection", docs...))
```

I am following the `TestInsertAndGet` test function but I am getting a nil pointer dereference panic because of the last line Insert.

Reply:
replace:
```go
docs := make([]*c.Document, 3)
```
with:
```go
docs := make([]*c.Document, 0)
```